source/iface/command-group-handler.rst

```diff
@@ -452,6 +452,8 @@ case the SYCL runtime will construct an instance of
 .. seealso:: :ref:`parallel_for_hierarchical`
 
 
+.. _hanlder_expl_mem_ops:
```

Comment:
There is a typo here.  Should it be `handler`... instead of `hanlder`... ?

Reply:
Yes, thanks a lot! Fixed in the last commit.